src/PrestaShopBundle/Resources/views/Admin/Sell/Order/Cart/view.html.twig

```diff
@@ -34,19 +34,35 @@
   {% endblock %}
 
   <div class="row">
-    <div class="col">
-      {% block cart_customer_information %}
-        {% include '@PrestaShop/Admin/Sell/Order/Cart/Blocks/View/customer_information.html.twig' %}
-      {% endblock %}
-    </div>
-    <div class="col">
+    <div class="col-md-6">
       {% block cart_order_information %}
         {% include '@PrestaShop/Admin/Sell/Order/Cart/Blocks/View/order_information.html.twig' %}
       {% endblock %}
     </div>
+    <div class="col-md-6">
+      {% block cart_customer_information %}
+        {% include '@PrestaShop/Admin/Sell/Order/Cart/Blocks/View/customer_information.html.twig' %}
+      {% endblock %}
+    </div>
   </div>
 
   {% block cart_summary %}
     {% include '@PrestaShop/Admin/Sell/Order/Cart/Blocks/View/cart_summary.html.twig' %}
   {% endblock %}
+
+  {% if cartView.cartSummary.cart_link is not empty %}
+    <div class="card" data-role="customer-information">
+      <h3 class="card-header">
+        <i class="material-icons">mail</i>
+        {{ 'Share this cart'|trans({}, 'Admin.Orderscustomers.Feature') }}
+      </h3>
+      <div class="card-body">
+        <p>{{ 'If you want to share this cart with a customer, send him the following link. After opening it, it will load the cart and all the customer information automatically. The customer can then finish the order in just a few clicks.'|trans({}, 'Admin.Orderscustomers.Feature') }}</p>
```

Comment:
```suggestion
        <p>{{ 'Send this link to your customer so they can review the order and submit payment.'|trans({}, 'Admin.Orderscustomers.Feature') }}</p>
```

Reply:
I would keep longer description, so the merchants know what the link will do. Can you make another version that will keep the original message?

Reply:
What information is missing in my suggestion? 

- The merchant can send the link to their customer
- Customer can review the order (make sure everything is right) 
- Customer can submit payment

Reply:
`Send this link to your customer so they can review the order and submit payment.` - I think it doesn't tell you what this link will do. Review the order how?

Reply:
What about: 

"Send this link to your customer so they can check the contents of their cart and their information before submitting payment." ?

Reply:
Julie I will write my thoughts and you try some another sentence that will involve everything. Sorry I complicate things. 😄😄

What I would like it to say:
- Who it's for - That it's for customer or a colleague. Or for example when the customer is on another device and lost access to that cart.
- What it will do - That it will load the cart and all information automatically on any device.
- That he will be able to review the order and finish the payment.

Reply:
=> Target: It could be a customer, a colleague, or maybe even someone else depending on the use case. It's difficult to tackle all use cases in one sentence. Maybe we could provide more information about this feature in the user guide?
=> Action: "Send this link to your customer **so they can review the contents of their cart and their information before submitting payment**". This implies that a page will load with the cart and all information. Don't you think?
=> "That he will be able to review the order and finish payment". This information is already included in my suggestion. 

Maybe you could add something about devices. For example:

"Thanks to this link, your customer can open the cart on their device and check the contents of the order and their details before submitting payment."